spec/factories/curator/filestreams/ereaders.rb

```diff
@@ -7,6 +7,16 @@
     file_set_type { 'Curator::Filestreams::Ereader' }
     file_name_base { Faker::Book.publisher }
     position { 1 }
+    hand_side { 'left' }
+    page_type { 'TOC' }
+    page_label { '3' }
```

Comment:
`hand_side`, `page_type`, and `page_label` would not be used with this FileSet type